web/components/roomConfig/FeedsConfig.tsx

```diff
@@ -58,14 +62,16 @@ const ConnectionConfiguration: FunctionComponent<ConnectionConfigurationProps<Se
         <InputField visible={true} label="Label" noPadding={true}>
             <input ref={labelRef} disabled={!canSave} type="text" value={existingConnection?.config.label} />
         </InputField>
-        <InputField visible={true} label="Template" noPadding={true}>
+        <InputField visible={onlyVisibleOnExistingConnection} label="Send a notice on read failure" noPadding={true}>
```

Comment:
It would be nice if these extra controls (the read failure & template sections) also appeared for the feed to be added (instead of only for already-added feeds) so that they can be set without having to add the feed first.  But that can be done in a later PR (if we even want that).

Reply:
It's basically intentional to avoid having the UI overwhelm the user. The create new feed UI took up so much verical space with this visible that I felt it would be better to show just the URL and Label and allow users to adjust to taste afterwards.